test/acceptance/helpers.js

```diff
@@ -0,0 +1,95 @@
+import express from 'express'
+import Nightmare from 'nightmare'
+import moment from 'moment'
+import cheerio from 'cheerio'
+import url from 'url'
+
+const {
+  ACCEPTANCE_TIMEOUT,
+  API_URL,
+  APP_URL,
+  METAPHYSICS_ENDPOINT,
+  POSITRON_URL
+} = process.env
+const [
+  FORCE_PORT,
+  GRAVITY_PORT,
+  METAPHYSICS_PORT,
+  POSITRON_PORT
+] = [
+  APP_URL,
+  API_URL,
+  METAPHYSICS_ENDPOINT,
+  POSITRON_URL
+].map((u) => url.parse(u).port)
```

Comment:
This double `[e, f, g, h] = [a, b, c, d].map(x => y)` seems a little too clever at a glance. What is happening right here?